Cargo.toml

```diff
@@ -78,8 +78,8 @@ getrandom = { version = "0.2", features = ["js"], optional = true }
 # When using the hax toolchain, we have more dependencies.
 # This is only required when doing proofs.
 [target.'cfg(hax)'.dependencies]
-hax-lib-macros = { version = "0.1.0-alpha.1", git = "https://github.com/hacspec/hax", branch = "main" }
-hax-lib = { version = "0.1.0-alpha.1", git = "https://github.com/hacspec/hax/", branch = "main" }
+hax-lib-macros = { git = "https://github.com/hacspec/hax", branch = "main" }
```

Comment:
Switching to `version = "0.1.0-alpha.1", git = "https://github.com/hacspec/hax/", branch = "main"` but maybe this will need to be a public crate anyway when we release?

Reply:
That's using a public crate when available.